tsconfig.json

```diff
@@ -3,6 +3,8 @@
     "outDir": "./dist/js",
     "module": "commonjs",
     "target": "es6",
+    "declaration": true,
+    "downlevelIteration": true,
```

Comment:
I am not sure we really need `downlevelIteration`. Project is target to es6, we already use class, spread operator and etc.

Reply:
yes, I previously set target to `es5`. so I add this option and update spread operator.